src/MultiDropdown.js

```diff
@@ -3,20 +3,78 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
-
-// HINT User CustomDropdown.js here
-
+import CustomDropdown from "./CustomDropdown";
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = (
+      {
+        "selectedValue1":null,
+        "selectedValue2":null,
+      }
+    );
+  }
+  onChange(selecetedValue,selectedElement) {
+    if(selecetedValue){
+      selecetedValue = selecetedValue.value;
+      if(selectedElement == "selectedValue1"){
+        this.setState({selectedValue1:selecetedValue});
+        this.setState({selectedValue2:null});
+      }
+      else if(selectedElement == "selectedValue2"){
+        this.setState({selectedValue2:selecetedValue});
+      }
+    }
+  }
+  getCities(){
+    if(this.state.selectedValue1){
+      let cityNames = countriesCities.getCities(this.state.selectedValue1).map(defaultValue=>{
+        return {
+          value:defaultValue,
+          label:defaultValue
+        }
+      });
+      return cityNames;
+    }
+    return [];
+  }
+  getCountries(){
+    let countryNames = countriesCities.getCountries().map(defaultValue=>{
```

Comment:
Do you have prettier set up in your environment? That will fix these spacing/indentation issues.

Reply:
I do have prettier setup in environment. Still it's not picking up the indentation. Looking into it.